projects/npm-tools/packages/npm-scripts/src/webpack/tweakWebpackConfig.js

```diff
@@ -45,6 +46,12 @@ async function tweakWebpackConfig(webpackConfigPath) {
 		mergeBabelLoaderOptions(webpackConfig)
 	);
 
+	createTempFile(
+		'webpack.config.json',
+		JSON.stringify(arrayConfig, null, '\t'),
```

Comment:
![attention-to-detail](https://user-images.githubusercontent.com/7074/110596620-cae88d80-817f-11eb-922b-740390714d45.gif)

(for the tabs)

Reply:
:joy: